lib/propshaft/resolver/dynamic.rb

```diff
@@ -12,9 +12,9 @@ def resolve(logical_path)
       end
     end
 
-    def read(logical_path)
+    def read(logical_path, options = {})
       if asset = load_path.find(logical_path)
-        asset.content
+        asset.content(**options)
```

Comment:
Is there a reason why you used `options` here instead of just `encoding` like in `assets.rb` and `static.rb`?

Reply:
I used an option hash to avoid duplicating the default value to this top-level method.

Alternative 1:

```
def read(logical_path, encoding: 'ASCII-8BIT')
  # ...
  asset.content(encoding: encoding)
  # ...
end
```

Alternative 2:

```
def read(logical_path, encoding: nil)
  # ...
  options = {}
  options[:encoding] = encoding if encoding.present?
  asset.content(**options)
  # ...
end
```

@brenogazzola Wdyt?